tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -843,13 +873,17 @@ class MklFusedBatchNormGradOp : public OpKernel {
       const size_t kScaleIndex = 2;     // index of scale tensor
       const size_t kMeanIndex = 3;      // index of saved_mean tensor
       const size_t kVarianceIndex = 4;  // index of saved_variance tensor
+      const size_t kReSpaceIndex = 5;   // index of reserved space 3 tensor
 
       const Tensor& diff_dst_tensor = MklGetInput(context, kDiffDstIndex);
       const Tensor& src_tensor = MklGetInput(context, kSrcIndex);
       const Tensor& scale_tensor = MklGetInput(context, kScaleIndex);
       const Tensor& saved_mean_tensor = MklGetInput(context, kMeanIndex);
       const Tensor& saved_variance_tensor =
           MklGetInput(context, kVarianceIndex);
+      const Tensor temp;
+      const Tensor& reserved_space_tensor =
+          (reserved_space) ? MklGetInput(context, kReSpaceIndex) : temp;
```

Comment:
Can we replace `temp` with `Tensor()`?
```suggestion
          (reserved_space) ? MklGetInput(context, kReSpaceIndex) : Tensor();
```

Reply:
Done